bosh-shell/startup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu
+
+if [ -z "${BOSH_ID_RSA:-}" ] || [ -z "${BOSH_IP:-}" ]; then
+  echo "WARNING: You need to set BOSH_ID_RSA and BOSH_IP. You will not be logged into Bosh"
+  exec bash
+fi
+
+echo "$BOSH_ID_RSA" | base64 -d > /tmp/bosh_id_rsa && chmod 400 /tmp/bosh_id_rsa
+
+ssh -fNT -L 25555:localhost:25555 \
```

Comment:
I'm seeing a warning `bind: Address not available` from this command. It seems that it's because it tried to listen on IPv6 localhost as well as IPv4, but this isn't available within the container. I can silence this warning by using the `-4` option to ssh.

Reply:
Good catch! We were wondering where that came from.

Reply:
Nice one. Fixed.